main.c

```diff
@@ -63,19 +63,31 @@ int main (int argc, char* argv[]) {
 	// the system without the need for another thread to join with the terminated thread
 	pthread_attr_setdetachstate(&attr, PTHREAD_CREATE_DETACHED);
 
+	// Start log analyzing thread
 	pthread_t piholelogthread;
 	if(pthread_create( &piholelogthread, &attr, pihole_log_thread, NULL ) != 0)
 	{
 		logg("Unable to open Pi-hole log processing thread. Exiting...");
 		killed = 1;
 	}
+	sleepms(100);
```

Comment:
Why is this sleep here? Why 100ms?

Reply:
I don't think this delay is needed at all.